pwndbg/commands/context.py

```diff
@@ -49,19 +61,17 @@ def context(*args):
     Accepts subcommands 'reg', 'code', 'stack', 'backtrace', and 'args'.
     """
     if len(args) == 0:
-        args = ['reg','code','stack','backtrace','args']
+        args = str(config_context_sections).split()
 
     args = [a[0] for a in args]
```

Comment:
I'm not sure it really matters to use a `list` versus a `genexpr` for a small number of elements